source/_redirects

```diff
@@ -2066,6 +2067,11 @@
 /integrations/switch.knx /integrations/knx#switch
 /integrations/weather.knx /integrations/knx#weather
 /integrations/zigbee /integrations/xbee
+/integrations/hassio/ /hassio
+/integrations/cloud.alexa/ /cloud/alexa
+/getting-started/automation-create-first/ /getting-started/automation
+/getting-started/automation-2/ /getting-started/automation
```

Comment:
@frenck Why do we use a `2` here? 🤔

Reply:
That is a pre-history URL, but still, it is referenced on some places.

Reply:
Zero results on the docs website, and only 11 links on the community forum topic from 2017 and 2018 🤔

Reply:
Yep, and google as well. There is no need to ditch it if we can redirect people. So this is fine.